corehq/apps/data_interfaces/models.py

```diff
@@ -661,6 +670,28 @@ def matches(self, case, now):
         return False
 
 
+class LocationFilterDefinition(CaseRuleCriteriaDefinition):
+
+    location_id = models.CharField(max_length=255)
+    include_child_locations = models.BooleanField(default=False)
+
+    def matches(self, case, now):
+        if case.owner_id:
+            # Checks if case belongs to location (hierarchy)
+
+            if not self.include_child_locations:
+                return case.owner_id == self.location_id
```

Comment:
As far as I know a case will belong to a location when `case.owner_id` == `location_id`, but I'm unsure whether or not I should consider `case.location_id`? In what context is `case.location_id` set if we use `case.owner_id` to denote case ownership? (maybe I should quickly verify my assumption)